src/LogicItLab/Salesforce/MapperBundle/WsdlValidator.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace LogicItLab\Salesforce\MapperBundle;
+
+use LogicItLab\Salesforce\MapperBundle\Annotation\AnnotationReader;
+use LogicItLab\Salesforce\MapperBundle\Exception\FileHandlerNotInitializedException;
+use Symfony\Component\Finder\Finder;
+
+class WsdlValidator
+{
+    /** @var AnnotationReader */
+    private $annotationReader;
+
+    /** @var string */
+    private $root;
+
+    /** @var string */
+    private $path;
+
+    /** @var string */
+    private $wsdlPath;
+
+    /**
+     * @param AnnotationReader $annotationReader
+     * @param string $root
+     * @param string $wsdlPath
+     * @codeCoverageIgnore
+     */
+    public function __construct(AnnotationReader $annotationReader, string $root, string $path, string $wsdlPath)
+    {
+        $this->annotationReader = $annotationReader;
+        $this->root = $root;
+        $this->path = $path;
```

Comment:
Since root and path are only used together doesn't it make more sense to have them as one "documentPath" variable? Then it also directly shows what the point of the variable is.

Reply:
in getAllClassNames I used them separately. Just the root is removed to obtain correct class name. Did I use incorrect name for them? Maybe can be more cleare

Reply:
Ah right yeah. I didn't see it properly. I think the names could maybe be a bit more clear because now it's not super very clear what path you want in it.